src/TorchSharp/TorchVision/Image.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+using static TorchSharp.torch;
+
+namespace TorchSharp.torchvision
+{
+    public static partial class io
+    {
+
+        public static IImager DefaultImager { get; set; } = new ImagerSharp();
+
+        public enum ImageFormat
+        {
+            PNG,
```

Comment:
>PNG [](http://example.com/codeflow?start=12&length=3)

Can we use the letter casing as we do in [https://docs.microsoft.com/en-us/dotnet/api/system.drawing.imaging.imageformat?view=dotnet-plat-ext-6.0#properties](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.imaging.imageformat?view=dotnet-plat-ext-6.0#properties)?